qpdf/qpdf.cc

```diff
@@ -1905,14 +1905,13 @@ static void do_show_pages(QPDF& pdf, Options& o)
     {
         dh.pushInheritedAttributesToPage();
     }
-    std::vector<QPDFObjectHandle> pages = dh.getAllPages();
+    std::vector<QPDFPageObjectHelper> pages = dh.getAllPages();
     int pageno = 0;
-    for (std::vector<QPDFObjectHandle>::iterator iter =
-             pages.begin();
+    for (std::vector<QPDFPageObjectHelper>::iterator iter = pages.begin();
          iter != pages.end(); ++iter)
     {
-        QPDFObjectHandle& page = *iter;
-        QPDFPageObjectHelper ph(page);
+        QPDFPageObjectHelper ph(*iter);
```

Comment:
Why the extra `QPDFPageObjectHelper` and not use `iter->getObjectHandle()` directly (or make `ph` a reference to *iter)?

Reply:
oh, editing error in updating the interface. Good catch, thanks.